### ISSUE TITLE
Cannot read property 'value' of null: this.matSelect.ngControl is null

### ISSUE THREAD

**Issue by macjohnny** [bithost-gmbh/ngx-mat-select-search#281]
@macjohnny @broekema41 
Hi :)
i'm just updating an app to angular 11 and bumbed the ngx-mat-select-search version from 3.0.3 to 3.1.3. With the new version I get null-references-failures, because `this.matSelect.ngControl` is null. This seems to be caused by the changes in `mat-select-search.component.ts`s `initMultipleHandling`. The previous access was just `this.matSelect.xxx` and the commit changed it to `this.matSelect.ngControl.xxx`. I reverted back to 3.0.3 and that versions works with angular 11. 
I haven't gotten the time yet to produce a minimal example to check, if i can reproduce the problem in a minimal environment though. 

best regards

_Originally posted by @Daishy in https://github.com/bithost-gmbh/ngx-mat-select-search/issues/279#issuecomment-728885214_

**Comment by macjohnny** [comment]
@Daishy thanks for reporting this issue.
the error occurs if there is neither an `ngModel` nor `formControl` directive connected to the `mat-select`.
as a quick workaround, you could just add `ngModel` without binding to any variable, 
```html
    <mat-select ngModel>
      <mat-option>
        <ngx-mat-select-search [formControl]="bankFilterCtrl"></ngx-mat-select-search>
      </mat-option>
    </mat-select>
```
I think I will release a new version that does not depend on `ngControl` being present.

**Comment by Daishy** [comment]
Thanks, i will try that. Currently there is neither in my setup, i just listened to the valueChange, so that checks out. 

And a quick thanks for the nice component, it's a default-install in nearly all angular-projects i'm managing :)

**Comment by macjohnny** [comment]
@Daishy a fix was released in version 3.1.4
